server/start.go

```diff
@@ -205,6 +219,7 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 	cmd.Flags().Uint32(FlagStateSyncSnapshotKeepRecent, 2, "State sync snapshot to keep")
 	cmd.Flags().Bool(FlagDisableIAVLFastNode, false, "Disable fast node for IAVL tree")
 	cmd.Flags().Int(FlagMempoolMaxTxs, mempool.DefaultMaxTx, "Sets MaxTx value for the app-side mempool")
+	cmd.Flags().Duration(FlagShutdownGrace, 0*time.Second, "On Shutdown, duration to wait for database to flush to disk")
```

Comment:
Can we reword this too?

Reply:
updated

Reply:
Thanks, you reworded the changelog, but can you reword the flag name as well? Then it's good to merge :)

Reply:
:) it looked so much the same.  updated!